namespaces/resource-namespace.md

```diff
@@ -0,0 +1,150 @@
+# Resource Namespace
+
+This document extends [Refract][] [MSON Namespace][] to enable Resource data structures.
+
+# About this Document
+
+This document conforms to RFC 2119, which says:
+
+> The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
+[MSON][] is used throughout this document.
+
+## Supporting Data Types
+
+### Href (string)
+
+The value of `Href` is to be used as a URI Template, as defined in RFC 6570.
+The value of the `Href` is a template used to determine the target URI of the related resource.
+The value of the instance property SHOULD be resolved as a URI-Reference per RFC 3986 and MAY be a relative reference to a URI.
+
+### Href Variables (Object Type)
+
+The definition is a MSON namespace `Object Type` where keys are respective URI Template variables.
+
+### Data Structure (MSON Element)
+
+Data structure definition using MSON namespace elements.
+
+## Namespace Elements
+
+### Resource (Element)
+
+The Resource representation with its available transitions and attributes.
+
+#### Properties
+
+- `element`: resource (string, fixed)
+- `attributes` (object)
+    - `href` (Href) - URI Template of this resource.
+    - `hrefVariables` (Href Variables) - Definition of URI Template variables used in the `href` property.
+
+- `content` (array)
+    - (Transition) - State transition available for this resource
+    - (Data Structure) - Data structure associated with the resource
```

Comment:
> Can a resource have multiple transitions and multiple data structures?

1. Yes, you can have as many `transitions` in the content's array as you like. 
2. Yes, you can have as many `data structures` in the content's array as you like.